migration/fmtp/CsvStringifyModified.js

```diff
@@ -265,7 +265,7 @@
            field = quote + field + quote;
          }
          newLine += field;
-       } else if (this.options.quotedEmpty || ((this.options.quotedEmpty == null) && line[i] === '' && this.options.quotedString)) {
+       } else if (this.options.quotedEmpty || ((this.options.quotedEmpty == null) && line[i] === '' && columns[i].indexOf('0000-00-00 00:00:00') === -1 && this.options.quotedString)) {
```

Comment:
The problem is that ColumnsDataArranger transforms dates to strings, so there's no other way (that I have found) to find if a field is a string or a date. A `Date` can not be quoted ("" is not a valid date) and an empty `String` must be quoted so that it is distinguished from NULL.

Maybe migrating to later CvsStringify, that has the concept of formatter, could lead to a cleaner code.